llvm/test/CodeGen/AArch64/GlobalISel/legalizer-info-validation.mir

```diff
@@ -674,6 +674,9 @@
 # DEBUG-NEXT: .. opcode {{[0-9]+}} is aliased to {{[0-9]+}}
 # DEBUG-NEXT: .. the first uncovered type index: 1, OK
 # DEBUG-NEXT: .. the first uncovered imm index: 0, OK
+# DEBUG-NEXT: G_FTAN (opcode {{[0-9]+}}): 1 type index, 0 imm indices
```

Comment:
It means that the AArch64 GlobalIsel legalizer does not say anything about G_FTAN.

Reply:
I know. There is no way to do a `G_FTAN` that isn't exposed across all backends.

Reply:
I am tracking changes to support arm64 here: https://github.com/farzonl/llvm-project/commit/aade6dde06f686e35f5c526cd915ba3b76380d16

It will have to come after this PR merges.